css/style.css

```diff
@@ -77,14 +131,11 @@ body {
     width: 100%;
 }
 
-#genotifyheader {
-    margin: 0px;
-    padding: 0px;
-}
-
 
 /* For label hiding */
 
 .hidden {
-    visibility: hidden;
-}
\ No newline at end of file
+    display: none;
+}
+
+.addlink {}
```

Comment:
This is actually used to denote divs whose content will also be linked to an external database - they just don't need any style changes as well. Easier than doing it with Id or name.

Reply:
I don't understand. Let's discuss this further.

Reply:
Take a look at the `displayData` function in `query.js`. I have to manually add the link elements to the appropriate div to link out to external databases, this rule is used to mark the divs where a link element must be added (or removed). The `textContent` of the div is used to dynamically create the full link.

For example:
`https://www.ncbi.nlm.nih.gov/gene/` is the static link to the database, but the actual gene in question (say CDK2) has an ID associated with it (1771) that's tacked on to the end of that string to create the full link.

https://www.ncbi.nlm.nih.gov/gene/1771

However, this only needs to be done for certain divs that contain IDs that will be used to link out to various databases. I'm using this rule as a way to mark them for this link creation (and removal) so that I don't have to comb through by div `id` or `name` instead. Perhaps there is a better solution, but this seemed like a straightforward way to achieve that goal.